src/libs/feature-previews-config.js

```diff
@@ -20,6 +20,13 @@ const featurePreviewsConfig = [
     groups: ['preview-wds-on-azure'],
     feedbackUrl: `mailto:dsp-analysis-journeys@broadinstitute.org?subject=${encodeURIComponent('Feedback on WDS UI')}`
   },
+  {
+    id: 'jupyterlab-gcp',
+    title: 'JupyterLab on GCP',
+    description: 'Enabling this feature will allow you to launch notebooks using JupyterLab in GCP workspaces.',
+    groups: ['preview-jupyterlab-gcp'],
+    feedbackUrl: `mailto:dsp-sue@broadinstitute.org?subject=${encodeURIComponent('Feedback on JupyterLab (GCP)')}`
+  },
```

Comment:
The reason for this second layer of feature flagging is that we need some documentation to support this since there are some limitations around notebook locking. Once that's rolled out, we'll include a link to it in the green banner and remove this global feature flag.